test/functional/wallet_backup.py

```diff
@@ -116,7 +116,7 @@ def erase_three(self):
     def restore_invalid_wallet(self):
         node = self.nodes[3]
         invalid_wallet_file = self.nodes[0].datadir_path / 'invalid_wallet_file.bak'
-        open(invalid_wallet_file, 'a', encoding="utf8").write('invald wallet')
+        open(invalid_wallet_file, 'a', encoding="utf8").write('invalid wallet')
```

Comment:
This is just a test, so should be OK to correct typo in the written contents to file test file?